src/plugins/intel_cpu/src/emitters/plugin/aarch64/debug_capabilities.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef DEBUG_CAPABILITIES_HPP
+#define DEBUG_CAPABILITIES_HPP
+
+#include <iostream>
+#include <cstdint>
+#include <arm_neon.h> // For SIMD support
+#include "openvino/util/ov_string_utils.hpp" // For ov::util::join
+
+class RegPrints {
+public:
+    static void print_gpr(jit_generator &gen, const uint64_t &reg_value, const char *reg_name) {
+        // Emit JIT code to print general-purpose register during runtime
+        gen.mov(gen.rdi, reg_value); // Move register value into rdi
+        gen.mov(gen.rsi, reinterpret_cast<uint64_t>(reg_name)); // Pass register name as argument
+        gen.call(reinterpret_cast<void(*)(const char*, uint64_t)>(print_runtime_gpr)); // Call runtime function

```

Comment:
The instruction `call` is missed on aarch64. Please take a look at the instruction [blr](https://developer.arm.com/documentation/ddi0602/2024-03/Base-Instructions/BLR--Branch-with-Link-to-Register-?lang=en). It calls a subroutine at an address in a register.

The registers `rdi` and `rsi` are also missed on aarch64. This naming is used on x64 platforms.

Also, as mentioned in the issue, the perfect example for this task is the implementation of `jit_power_static_emitter` which generates code to call `std::powf` in runtime. Please take a look at this [impl](https://github.com/openvinotoolkit/openvino/blob/91a5518347a5e4e55f940b6e3c9b8f00626f09ff/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp#L1769-L1793).